build.cake

```diff
@@ -93,6 +95,14 @@ Task("Restore")
     });
 
 
+
+Task("Boots")
```

Comment:
I couldn't get the dotnet CLI version to work on mac so I just used cake

I think maybe this would fix the dotnet CLI
https://github.com/dotnet/cli/issues/9114#issuecomment-494226139